### PR TITLE
Bump pyjwt, cryptography, social-auth-core

### PR DESCRIPTION
**Pull request by nicobav**
Needed to bump social-auth-core (🤞 ) to get latest pyjwt. Previously used version of pyjwt had some security issues.

Refs RAT-18

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -147,8 +147,6 @@ icalendar==5.0.4
     # via -r requirements.in
 idna==3.4
     # via requests
-importlib-metadata==6.0.0
```
Comment (charn) [inline requirements.txt:150]: This should not get removed. You've probably run `pip-compile` with Python=<3.9.

File: requirements.txt [hunk 2]
```diff
@@ -312,8 +309,6 @@ webencodings==0.5.1
     # via bleach
 whitenoise==6.4.0
     # via -r requirements.in
-zipp==3.15.0
```
Comment (charn) [inline requirements.txt:315]: This should also not get removed.

File: requirements.txt [hunk 3]
```diff
@@ -274,11 +272,10 @@ six==1.16.0
     #   pyjwkest
     #   python-dateutil
     #   social-auth-app-django
-    #   social-auth-core
     #   url-normalize
 social-auth-app-django==4.0.0
     # via -r requirements.in
-social-auth-core==4.0.2
+social-auth-core==4.0.3
```
Comment (charn) [inline requirements.txt:278]: I wonder if we could bump this even more. Though I guess this is enough since it has the support for the new pyjwt.

Comment (nicobav) [reply requirements.txt:278]: Perhaps we could. I just wanted to play sure. What we get if we upgrade higher, that I don't know.

Comment (nicobav) [reply requirements.txt:278]: So if we stay for now at this 4.0.3?

File: requirements.txt (hunk 1 above)
Comment (nicobav) [reply requirements.txt:150]: Nicely catched 🎣

File: requirements.txt (hunk 3 above)
Comment (charn) [reply requirements.txt:280]: Fine by me! :+1:

**Review by charn — APPROVED** [review]
LGTM! :+1: :shipit:

**Pull request merged**